packages/clients/src/aptos/aptos.ts

```diff
@@ -38,8 +36,8 @@ function inferNetworkFromKey(accessKey: string): Network | undefined {
  */
 export function createAptosClient(
   accessKey: string,
-  url: string = NODE_REST_URL,
-  indexerUrl: string = NODE_INDEXER_URL,
+  url: string = NodeRestUrls.us1,
+  indexerUrl: string = NodeIndexerUrls.us1,
```

Comment:
I just had another thought: instead of requiring people to specify any non-US1 urls, can we infer that from `accessKey` given our naming scheme, similar to `inferNetworkFromKey` (which we also need to update btw)? It's safe to assume any non-inferable keys to belong to `us1` since it's the same behavior as today if you don't specify urls. Something like this would preserve the current function signature and the ability to selectively override urls:

```ts
export function createAptosClient(
  accessKey: string,
  url?: string,
  indexerUrl?: string,
): Aptos {
  if (!url || !indexerUrl) {
    const region = inferRegionFromKey(accessKey) ?? "us1";
    if (!url) url = NodeRestUrls[region];
    if (!indexerUrl) indexerUrl = NodeIndexerUrls[region];
  }
  ...
}
```

Reply:
apac1/eu1 old keys are not prefixed by region

Reply:
> It's safe to assume any non-inferable keys to belong to us1 since it's the same behavior as today if you don't specify urls.

Reply:
![2025-04-10-194404_964x358_scrot](https://github.com/user-attachments/assets/02eeaa29-91bd-461e-ad1c-d17cb67a5f7f)

it doesn't work, return type of `inferRegionFromKey` is a subtype of `keyof NodeRestUrl`